await.js

```diff
@@ -21,11 +21,23 @@ const readFile = (options, filename) =>
 
 async function main() {
   const index = path.join(process.argv[2], 'index.txt');
-  const data = await readFile({encoding: 'utf8'}, index);
+  let data;
+  try {
+    data = await readFile({encoding: 'utf8'}, index);
+  } catch (err) {
+    process.stderr.write(String(err) + '\n');
+    process.exit(1);
+  }
```

Comment:
One would think so! In fact, the program exits immediately with an exit code of 0, without writing anything to stdout or to stderr:

```console
$ node --harmony await.compiled.js input
FOO
BAR
BAZ

$ node --harmony await.compiled.js XXX

$ echo $?
0
```

This may be a bug in Babel, I don't know.